.cirrus.yml

```diff
@@ -29,6 +29,7 @@ global_task_template: &GLOBAL_TASK_TEMPLATE
   depends_releases_cache:
     folder: "/tmp/cirrus-ci-build/releases"
   merge_base_script:
+    - if [ "$CIRRUS_PR" = "" ]; then exit 0; fi
```

Comment:
does this exit or continue?

Would be good to link to a successful build, so that it can be checked.

Reply:
It continues with the next script.

Here's how I tested it: https://cirrus-ci.com/task/4944025117523968